views/new_splash.pug

```diff
@@ -6,42 +6,27 @@ html(lang='en')
     link(rel='stylesheet', href='assets/new_splash.css')
     script(src='https://www.google.com/recaptcha/api.js')
   body
-    // Generator: Adobe Illustrator 21.1.0, SVG Export Plug-In
-    img.logo(src='assets/logo.svg')
-    main
-      .media
-        svg.media-figure(width='123px', height='123px', viewbox='0 0 123 123', version='1.1', xmlns='http://www.w3.org/2000/svg', xmlns:xlink='http://www.w3.org/1999/xlink')
-          path#Shape(d='M79.3,8.1 C77.4,2.4 71.3,-0.7 65.6,1.1 C59.9,3 56.8,9.1 58.6,14.8 L86.7,101.2 C88.6,106.5 94.4,109.5 99.9,107.9 C105.7,106.2 109.2,100.1 107.3,94.5 C107.3,94.3 79.3,8.1 79.3,8.1 Z', fill='#ECB32D')
-          path#Shape(d='M35.8,22.2 C33.9,16.5 27.8,13.4 22.1,15.2 C16.4,17.1 13.3,23.2 15.1,28.9 L43.2,115.3 C45.1,120.6 50.9,123.6 56.4,122 C62.2,120.3 65.7,114.2 63.8,108.6 C63.8,108.4 35.8,22.2 35.8,22.2 Z', fill='#63C1A0')
-          path#Shape(d='M114.7,79.6 C120.4,77.7 123.5,71.6 121.7,65.9 C119.8,60.2 113.7,57.1 108,58.9 L21.5,87.1 C16.2,89 13.2,94.8 14.8,100.3 C16.5,106.1 22.6,109.6 28.2,107.7 C28.4,107.7 114.7,79.6 114.7,79.6 Z', fill='#E01A59')
-          path#Shape(d='M39.5,104.1 C45.1,102.3 52.4,99.9 60.2,97.4 C58.4,91.8 56,84.5 53.5,76.7 L32.8,83.4 L39.5,104.1 Z', fill='#331433')
-          path#Shape(d='M83.1,89.9 C90.9,87.4 98.2,85 103.8,83.2 C102,77.6 99.6,70.3 97.1,62.5 L76.4,69.2 L83.1,89.9 Z', fill='#D62027')
-          path#Shape(d='M100.5,36.1 C106.2,34.2 109.3,28.1 107.5,22.4 C105.6,16.7 99.5,13.6 93.8,15.4 L7.4,43.5 C2.1,45.4 -0.9,51.2 0.7,56.7 C2.4,62.5 8.5,66 14.1,64.1 C14.3,64.1 100.5,36.1 100.5,36.1 Z', fill='#89D3DF')
-          path#Shape(d='M25.4,60.5 C31,58.7 38.3,56.3 46.1,53.8 C43.6,46 41.2,38.7 39.4,33.1 L18.7,39.8 L25.4,60.5 Z', fill='#258B74')
-          path#Shape(d='M68.9,46.4 C76.7,43.9 84,41.5 89.6,39.7 C87.1,31.9 84.7,24.6 82.9,19 L62.2,25.7 L68.9,46.4 Z', fill='#819C3C')
-        .media-body
-          h2
-            | Join the Slack community of 
-            b.total #{total} 
-            | triangle-area designers. (
-            b.active #{active}
-            |  currently online) 
-          p
-            | Triangle Designers is a resource to connect and inspire digital and
-            | print designers throughout North Carolina’s Triangle.
-          form#invite
-            input(type='email' name='email' placeholder='email@example.com' autofocus='autofocus')
-            button Get my Invite
-            .g-recaptcha(data-sitekey=gcaptcha_sitekey)
```

Comment:
@toobulkeh I'd prefer to remove this until we see a need, but I'm not sure what all needs to happen to pull it out.